esrally/mechanic/launcher.py

```diff
@@ -371,48 +326,47 @@ def _set_env(self, env, k, v, separator=' '):
             else:  # merge
                 env[k] = v + separator + env[k]
 
-    def _start_process(self, env, node_name, binary_path):
+    @staticmethod
+    def _start_process(binary_path):
         if os.geteuid() == 0:
             raise exceptions.LaunchError("Cannot launch Elasticsearch as root. Please run Rally as a non-root user.")
         os.chdir(binary_path)
         cmd = ["bin/elasticsearch"]
-        return _start(subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.DEVNULL, env=env), node_name)
+        cmd.extend(["-d", "-p", "pid"])
+        proc = subprocess.Popen(cmd,
```

Comment:
At this time we still need to pass the environment otherwise telemetry devices that rely on `ES_JAVA_OPTS` will not work anymore.